tests/Kernel/RdfSkosGraphServiceTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\rdf_skos\Kernel;
+
+/**
+ * Tests the RDF SKOS entities.
+ */
+class RdfSkosGraphServiceTest extends RdfSkosKernelTestBase {
+
+  /**
+   * Test that op_skos_setup service sets the correct config.
+   */
+  public function testGraphSetupService(): void {
+    /** @var \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager */
+    $entity_type_manager = $this->container->get('entity_type.manager');
+
+    // Prepare the graph data.
+    $info = $this->getTestGraphInfo($_ENV['SIMPLETEST_BASE_URL'], 'phpunit');
+    $graphs = [];
+
+    foreach ($info as $name => $item) {
+      $graphs[$name] = $item['uri'];
+    }
+
+    // Call the service to import our data.
+    \Drupal::service('rdf_skos.op_skos_setup')->setup($graphs);
+
+    // Assert data was saved.
+    $ids = $entity_type_manager->getStorage('skos_concept_scheme')->getQuery()
```

Comment:
We shouldn't check the loading, but that the config has been updated correctly. The loading part is done in `\Drupal\Tests\rdf_skos\Kernel\RdfSkosEntitiesKernelTest::testMultipleGraphs()`. Let's just take the config and check that the values have been added.
Then, let's try to add again the same graph and check the config again to see that no duplicates are added (a part of our code does this).

Reply:
refactored